Extension/tslint.json

```diff
@@ -1,31 +1,184 @@
 {
     "rules": {
+        "adjacent-overload-signatures": true,
+        "align": true,
+        "array-type": false,
+        "arrow-parens": false,
+        "arrow-return-shorthand": true,
+        "await-promise": false,
+        "ban-comma-operator": true,
+        "binary-expression-operand-order": true,
+        "callable-types": true,
+        "class-name": true,
+        "comment-format": true,
+        "completed-docs": false,
+        "no-console": false,
         "curly": true,
+        "cyclomatic-complexity": false,
+        "deprecation": false,
+        "encoding": true,
+        "eofline": true,
         "ext-variable-name": [
             true,
-            ["class", "pascal"],
-            ["function", "camel"]
+            [
+                "class",
+                "pascal"
+            ],
+            [
+                "function",
+                "camel"
+            ]
         ],
-        "file-header": [true, ".*"],
-        "indent": [true, "spaces", 4],
+        "file-header": [
+            true,
+            ".*"
+        ],
+        "forin": false,
+        "import-spacing": true,
+        "indent": [
+            true,
+            "spaces",
+            4
+        ],
+        "interface-name": false,
+        "interface-over-type-literal": false,
+        "jsdoc-format": false,
+        "label-position": true,
+        "linebreak-style": false,
+        "match-default-export-name": true,
+        "member-access": false,
```

Comment:
Not a fan of this being `false`. Makes it harder to read. 

https://palantir.github.io/tslint/rules/member-access/